ovn-org/ovn-kubernetes
# unit tests for the annotation related functions in pkg/util

Signed-off-by: Vishwanath Jayaraman <vjayaram@redhat.com>



**- What this PR does and why is it needed**

The PR adds unit tests (code refactor is not the scope for this PR) for most of the functions defined in the following files:

    - pkg/util/node_annotations.go 
    - pkg/util/pod_annotation.go
    -pkg/util/subnet_annotations.go


**- Special notes for reviewers**



**- How to verify it**

Pull this PR into your workspace
Navigate to go-controller/pkg/util directory
execute below commands
go test -coverprofile cp.out
go tool cover -html=cp.out
select files node_annotations.go, pod_annotation.go, subnet_annotations.go from drop down to verify coverage

**- Description for the changelog**
